pkg/node/kubelet.go

```diff
@@ -116,6 +116,17 @@ func (s *Kubelet) configure(cfg *config.MicroshiftConfig) error {
 func (s *Kubelet) Run(ctx context.Context, ready chan<- struct{}, stopped chan<- struct{}) error {
 	defer close(stopped)
 
+	// run readiness check
+	go func() {
+		healthcheckStatus := util.RetryInsecureHttpsGet("https://127.0.0.1:10248/healthz")
+		if healthcheckStatus != 200 {
+			logrus.Fatalf("Kube-controller-manager failed to starttttttt")
+		}
+
+		logrus.Infof("%s is readyyy", s.Name())
```

Comment:
Remove repeated "y"s

Reply:
Sorry I added this for easy grep search in microshift logs for the sake of debugging. The e2e are failing and I was trying to debug it. I will remove it.